bgpd/bgp_route.c

```diff
@@ -6324,6 +6329,16 @@ static void route_vty_out_route(struct prefix *p, struct vty *vty,
 				vty, "%s/%d",
 				inet_ntop(p->family, &p->u.prefix, buf, BUFSIZ),
 				p->prefixlen);
+		else {
+			json_object_string_add(json, "prefix",
+						inet_ntop(p->family,
+							&p->u.prefix, buf,
+							BUFSIZ));
+			json_object_int_add(json, "prefixLen", p->prefixlen);
+			sprintf(buf2, "%s/%d",
```

Comment:
prefix2str?  Why are we building this by hand?

Reply:
Will change